apps/api/src/validation/validation.service.ts

```diff
@@ -0,0 +1,62 @@
+import Ajv from 'ajv';
+import { Inject, Injectable } from '@nestjs/common';
+import { Field, TildaData, Validator } from '../models';
+import { ValidatorFactory } from './validator-factory';
+import { ValidationResult } from './models/validation-result';
+
+@Injectable()
+export class ValidationService {
+  constructor(
+    @Inject('ValidatorFactory')
+    private readonly validatorFactory: ValidatorFactory,
+  ) {}
+
+  private addValidatorToSchema(
+    schema: any,
+    validator: Validator,
+    fieldName: string,
+  ): void {
+    const { factory, params } = validator;
+    const customValidator = this.validatorFactory.getValidator(factory);
+
+    if (customValidator) {
+      schema.properties[fieldName] = customValidator.getValidator(params);
+    }
```

Comment:
This will reset the first validation with the second one. We should keep both validators, for example, regex and not empty validators should not override each other.

Reply:
also need new test cases for that scenario.

Reply:
also naming as "applyValidatorToSchema" is more suitable